testsuite/tests/singlecluster/observability/conftest.py

```diff
@@ -0,0 +1,93 @@
+"""Conftest for observability tests"""
+
+import time
+
+import backoff
+import pytest
+from openshift_client import selector
+
+
+@pytest.fixture(scope="module")
+def authorization():
+    """Override the authorization fixture to prevent the creation of an AuthPolicy"""
+    return None
+
+
+@pytest.fixture(scope="module")
+def rate_limit():
+    """Override the rate limit fixture to prevent the creation of an RateLimitPolicy"""
+    return None
```

Comment:
If you look at `pytest --setup-plan`, you'll see that `commit` fixture is executed either way because it is on autouse. And it's doing nothing because it supposed to commit authpolicy and ratelimitpolicy, but you set them as None here. 

Just override commit fixture to do nothing here, instead of overwriting both policies. That way 1 fixture will be executed instead of 3. Also you can put some other fixtures, you don't know where to request at, as dependencies to the new `commit`